### ISSUE TITLE
log_rotate_count configuration not working with include_config_file

### ISSUE THREAD

**Issue by mrfrostfire** [processone/ejabberd#1650]
> What version of ejabberd are you using?

16.12, 17.03

> What operating system (version) are you using?

RHEL

> How did you install ejabberd (source, package, distribution)?

package

> What did not work as expected? Are there error messages in the log? What
> was the unexpected behavior? What was the expected result?

When setting log_rotate_count to 14, it's still working as if it was set to 1, only keepting ejabberd.log and ejabberd.log.0.

**Comment by badlop** [comment]
I've tried with ejabberd 17.03 compiled from git, and that option is used correctly: it renames the log files up to 14.

Try to set an invalid value, like this:
> log_rotate_count: aaa

Then, when you start ejabbed, it should complain like this:
> 15:51:06.015 [error] ignoring option 'log_rotate_count' with invalid value: aaa

Do you see that error message?

**Comment by mrfrostfire** [comment]
We changed the configuration to log_rotate_count: aaa as you requested, and we did see the errormessage in the logs. We then changed the log_rotate_count back to 14, set the log_rotate_size to 1024 to force rotation to happen faster. Then stopped the services (we use 2 in a cluster setup), removed all logs, then started up the services again. The error was gone but the logfile keeps growing beyond the limit.
If it makes any difference, we use 2 services in a cluster setup, where both are using the same configuration in a config file on a mapped samba share. The configurationfile is references using a single line of include_config_file.
